ServiceWebsite/ServiceWebsite.Common/RegExValidators.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace ServiceWebsite.Common
+{
+    public static class RegExValidators
+    {
+        public static bool ValidateForCapsAndUnderscore(string key)
```

Comment:
You can inline this and use only one static instance for the regex to avoid recompilation. That way you can name it more specifically too, like ValidAnswerKeyRegex or so

Reply:
Changes are addressed